tests/backup_helper.go

```diff
@@ -4920,6 +4948,41 @@ func CreateRuleForBackupWithMultipleApplications(orgID string, appList []string,
 	return preRuleName, postRuleName, nil
 }
 
+type VMRuleType string
+
+const (
+	Freeze   VMRuleType = "freeze"
+	Unfreeze VMRuleType = "unfreeze"
+)
+
+func CreateRuleForVMBackup(ruleName string, vms []kubevirtv1.VirtualMachine, ruleType VMRuleType, ctx context1.Context) error {
```

Comment:
Why can't we have the rukes as per the name of the vm?

Reply:
You can give any name to the rule. There are cases where there are VMs with same name in different namespaces.
So it is left to the caller to give any name required for the test case.